sos/report/plugins/aap_controller.py

```diff
@@ -56,13 +56,16 @@ def setup(self):
             "/var/lib/awx/venv/awx/bin/pip freeze -l",
             "/var/lib/awx/venv/ansible/bin/pip freeze",
             "/var/lib/awx/venv/ansible/bin/pip freeze -l",
-            "tree -d /var/lib/awx",
-            "ls -ll /var/lib/awx",
-            "ls -ll /var/lib/awx/venv",
-            "ls -ll /etc/tower",
             "umask -p",
         ])
 
+        self.add_dir_listing([
+            '/var/lib/awx',
+            '/var/lib/awx/venv',
+            '/etc/tower',
+        ])
```

Comment:
Missing `recursive=False`?

(should `recursive` have default value `True` or `False`? I would expect `False` since that follows default `ls` behaviour and I would expect it as a human-default thinking of "list directory" as well; also since the patch replaces 38 instances of recursive listings and 32 non-recursive ones, there is no need to have such strong preference to the "recursive is far more used default")

Reply:
> (should `recursive` have default value `True` or `False`? I would expect `False` since that follows default `ls` behaviour and I would expect it as a human-default thinking of "list directory" as well; also since the patch replaces 38 instances of recursive listings and 32 non-recursive ones, there is no need to have such strong preference to the "recursive is far more used default")

When I was first reviewing the PR I was thinking the same, and missed the `recursive=True` as the default in the function definition, and had put in ~10 comments before abandoning it. I would agree with @pmoravec here, I think having `recursive=False` should be default and we use `recursive=True` where needed